ui/components/ui/nft-info-setApprovalForAll/nft-info-setApprovalForAll.js

```diff
@@ -0,0 +1,108 @@
+import React, { useContext } from 'react';
+import PropTypes from 'prop-types';
+import { I18nContext } from '../../../contexts/i18n';
+import Box from '../box';
+import { Text, Icon, ICON_NAMES, ICON_SIZES } from '../../component-library';
+import {
+  BackgroundColor,
+  Color,
+  DISPLAY,
+  TextVariant,
+} from '../../../helpers/constants/design-system';
+import Tooltip from '../tooltip/tooltip';
+import NftCollectionImage from '../nft-collection-image/nft-collection-image';
+
+export default function NftInfoSetApprovalForAll({
+  assetName,
+  tokenAddress,
+  total,
+  isERC721,
+}) {
+  const t = useContext(I18nContext);
+
+  return (
+    <Box
+      display={DISPLAY.FLEX}
+      className="nft-info-setApproveForAll"
+      backgroundColor={BackgroundColor.backgroundAlternative}
+    >
+      <Box marginTop={4} marginBottom={4} marginLeft={4}>
```

Comment:
I think we can reduce some of the html here by not having the wrapper use flex box and using padding at the top layer

```suggestion
    <Box
      className="nft-info-setApproveForAll"
      backgroundColor={BackgroundColor.backgroundAlternative}
      padding={4}
    >
```